templates/al2023/provisioners/bb-devmapper-create.sh

```diff
@@ -1,32 +1,46 @@
 #!/bin/bash
 set -ex
 
-DATA_DIR=/var/lib/containerd/io.containerd.snapshotter.v1.devmapper
+sudo dnf install -y git make
+
+# Block device to use for devmapper thin-pool
+BLOCK_DEV=/dev/sdf
 POOL_NAME=devpool
+VG_NAME=containerd
+
+# Install container-storage-setup tool
+git clone https://github.com/projectatomic/container-storage-setup.git
+cd container-storage-setup/
+sudo make install-core
+echo "Using version $(container-storage-setup -v)"
+
+cd ../
+rm -rf container-storage-setup
 
-sudo mkdir -p ${DATA_DIR}
-# Create data file
-sudo touch "${DATA_DIR}/data"
-sudo truncate -s 100G "${DATA_DIR}/data"
+# Create configuration file
+# Refer to `man container-storage-setup` to see available options
+sudo tee /etc/sysconfig/containerd-storage-setup << EOF
+DEVS=${BLOCK_DEV}
+VG=${VG_NAME}
+CONTAINER_THINPOOL=${POOL_NAME}
+EOF
 
-# Create metadata file
-sudo touch "${DATA_DIR}/meta"
-sudo truncate -s 40G "${DATA_DIR}/meta"
+# Run the script
+sudo container-storage-setup
 
-# Allocate loop devices
-DATA_DEV=$(sudo losetup --find --show "${DATA_DIR}/data")
-META_DEV=$(sudo losetup --find --show "${DATA_DIR}/meta")
+sudo tee /usr/lib/systemd/system/containerd-storage-setup.service << EOF
+[Unit]
+Description=Containerd Storage Setup
+After=cloud-init.service
+Before=containerd.service
 
-sudo dnf -y install bc
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/container-storage-setup
+EnvironmentFile=-/etc/sysconfig/containerd-storage-setup
```

Comment:
just curious what the `-` syntax is in `-/etc/sysconfig/containerd-storage-setup`?